lib/player_type/player_human.rb

```diff
@@ -6,7 +6,7 @@ def initialize(questions)
 
   def take_turn(game_logic)
     valid_options = game_logic.open_spaces.map(&:to_s)
```

Comment:
@Cristian-Baeza 
I didn't catch this before. But you should no longer be converting the numbers to_s.

```ruby
valid_options = game_logic.open_spaces
```